sdk/packages/core/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @omni-network/core
 
+## 0.4.0
```

Comment:
Does it need to be a v0.4.0? If there's no breaking change, that could be added as v0.3.x, no?